unity-renderer/Assets/UnityGLTF/Scripts/Editor/GLTFImporter.cs

```diff
@@ -132,54 +132,54 @@ public override void OnImportAsset(AssetImportContext ctx)
                 var meshFilters = gltfScene.GetComponentsInChildren<MeshFilter>();
                 var vertexBuffer = new List<Vector3>();
                 meshes = meshFilters.Where(mf => mf.sharedMesh != null)
-                                    .Select(mf =>
-                                    {
-                                        var mesh = mf.sharedMesh;
+                    .Select(mf =>
+                    {
+                        var mesh = mf.sharedMesh;
```

Comment:
It's ok to use linq in the GLTFImporter? Won't it cause spikes? If not, just ignore this

Reply:
GLTFImporter is only used in editor-mode, so it shouldn't be a problem